vs/client/src/tasks.ts

```diff
@@ -1,58 +1,60 @@
-import { CustomExecution, EventEmitter, ExtensionContext, Pseudoterminal, Task, TaskDefinition, TaskScope, TerminalDimensions, WorkspaceFolder, tasks, workspace } from 'vscode';
-import { generateBuildFile } from './requests';
+import { CustomExecution, EventEmitter, ExtensionContext, Pseudoterminal, Task, TaskDefinition, TaskScope, TerminalDimensions, WorkspaceFolder, tasks } from 'vscode';
+import { LanguageClientManager } from './languageClientManager';
 
-interface SourceOrbitTask extends TaskDefinition {
-	builder: "bob"|"make"|"json";
-}
+export namespace SourceOrbitTask {
+	interface SourceOrbitTask extends TaskDefinition {
+		builder: "bob" | "make" | "json";

```

Comment:
what about this new imd thing?

Reply:
Good catch. I will add that here.

@worksofliam How are these tasks run? Are these something that should be in the list of runnable tasks?
![image](https://github.com/user-attachments/assets/0f7a52f4-9e3a-4554-a85c-28f57f109e97)